cumulusci.yml

```diff
@@ -608,29 +582,41 @@ flows:
             4:
                 task: publish_learner_portal
 
-    run_unmanaged_unit_tests:
+    qa_org_2gp_ftests:
+        group: "Org Setup"
+        description: Set up an org and run functional tests as a QA environment using a second-generation package
+        steps:
+            1:
+                flow: qa_org_2gp
+            2:
+                flow: run_functional_tests
+
+    run_functional_tests:
```

Comment:
One only concern here is that this will deploy all the unpackaged test metadata to the `default` scratch org. Does that "pollute" the development environment since we want it to be as close as the customer environment?

Reply:
so ideally we should have function_test_org and unpackaged_test org like:

- function_test(unpackaged)_org
-- qa_org_2gp
-- deploy metadata for a certain test type

And then we should have run_functional_test and run_unpackaged_test "only" running the test.

Reply:
One of our targets for releng is to make the flow an all-in-one, similar to ci-feature. We can individually run run_functional_tests on any managed org.